docs/src/refman/display.txt

```diff
@@ -679,3 +679,71 @@ or resets the system back to the default state
 (the state at program start) if false is passed.
 It returns true if the state was set successfully,
 otherwise false.
+
+
+
+## Clipboard
+
+With the clipboard API of Allegro, text can be copied from and to the clipboard.
+Currentlly, only UTF8 encoded text is supported, and it only works on Linux,
+Windows and, OSX . Other platforms such as Android and IOS are currently
+unsupported.
+
+
+### API: al_get_clipboard_text
+
+This function returns a pointer to a string, allocated with [al_malloc] with
+the text contents of the clipboard if available. If not text is available on the
+clipboard then this function returns NULL. You must call [al_free] on the
+returned pointer when you don't need it anymore.
+
+Beware that text on the clipboard on Windows may be in Windows format, that is,
+it may have carriage return newline combinations for the line endings in stead
+of regular newlines for the line endings on Linux or OSX.
+
+Since: 5.1.12
+
+See also: [al_get_clipboard_ustr], [al_set_clipboard_text],
+          [al_set_clipboard_ustr], [al_clipboard_has_text]
+
+### API: al_get_clipboard_ustr
+
+This function works the same as [al_get_clipboard_text] but returns a pointer to
+an [ALLEGRO_USTR] in stead of a char pointer. The result of this function must
+be freed with [al_ustr_free] when you don't need it anymore.
+
+Since: 5.1.12
+
+See also: [al_get_clipboard_ustr], [al_set_clipboard_text],
+          [al_set_clipboard_ustr], [al_clipboard_has_text]
+
+
+### API: al_set_clipboard_text
+
+This function pastes the text given as an argument to the clipboard.
+
+Since: 5.1.12
+
+See also: [al_get_clipboard_ustr], [al_get_clipboard_text],
+          [al_set_clipboard_ustr]
+
+
+### API: al_set_clipboard_ustr
+
+This function works the same as [al_set_clipboard_text] but takes a pointer to
+an [ALLEGRO_USTR] in stead of a char pointer.
+
+
+Since: 5.1.12
+
+See also: [al_get_clipboard_ustr], [al_get_clipboard_text],
+          [al_get_clipboard_text]
+
+### API: al_clipboard_has_text
+
+This function returns true if the clipboard has text available, false if not.
```

Comment:
Going to play devil's advocate on that one: to fully preserve the meaning you'd have to change the text to "returns true only if".  I tend to document bool returns the same as in the text above.